crates/hyperswitch_connectors/src/connectors.rs

```diff
@@ -1,2 +1,3 @@
+pub mod fiserv;
 pub mod helcim;
-pub use self::helcim::Helcim;
+pub use self::{fiserv::Fiserv, helcim::Helcim};
```

Comment:
Nit: Add a newline between the module declarations and the re-exports.

Reply:
Done